### PR TITLE
fcoe: rename rd.nofcoe to rd.fcoe

### PR DESCRIPTION
**Pull request by lnykryn**
fcoe: rename rd.nofcoe to rd.fcoe
    
    The current name of this bool is kinda stupid. Based on the manpage
    setting it to 0 turns off fcoe, which means that nofcoe=1 should mean
    that it is on.
    
    Let's just do the same thing as with rd.lvm=0, rd.luks=0,....

### CODE REVIEW

File: modules.d/95fcoe/lldpad.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-if getargbool 0 rd.nofcoe ; then
-	info "rd.nofcoe=0: skipping lldpad activation"
+if ! getargbool 1 rd.fcoe -d rd.nofcoe ; then
```
Comment (haraldh) [inline modules.d/95fcoe/lldpad.sh:3]: Shall we follow the wrong documentation with the deprecated option or the correct one with:

```suggestion
if ! getargbool 1 rd.fcoe -d -n rd.nofcoe ; then
```

**Review by haraldh — APPROVED** [review]

File: modules.d/95fcoe/parse-fcoe.sh [hunk 2]
```diff
@@ -13,8 +13,8 @@
 # fcoe=eth0:nodcb:vn2vn
 # fcoe=4a:3f:4c:04:f8:d7:nodcb:fabric
 
-if getargbool 0 rd.nofcoe ; then
-	info "rd.nofcoe=0: skipping fcoe"
+if ! getargbool 1 rd.fcoe -d rd.nofcoe ; then
```
Comment (haraldh) [inline modules.d/95fcoe/parse-fcoe.sh:16]: Shall we follow the wrong documentation with the deprecated option:

```suggestion
if ! getargbool 1 rd.fcoe -d -y rd.nofcoe ; then
```

or the correct one with:

```suggestion
if ! getargbool 1 rd.fcoe -d -n rd.nofcoe ; then
```

**Review by haraldh — CHANGES_REQUESTED** [review]

File: modules.d/95fcoe/parse-fcoe.sh (hunk 2 above)
Comment (johannbg) [reply modules.d/95fcoe/parse-fcoe.sh:16]: The correct one I would say.

**Comment by johannbg** [comment]
@lnykryn you need to fix your commit message to pass the commisery check, Change it to fix(fcoe): rename rd.nofcoe to rd.fcoe and it should pass the check.

File: modules.d/95fcoe/parse-fcoe.sh (hunk 2 above)
Comment (lnykryn) [reply modules.d/95fcoe/parse-fcoe.sh]: I would be for that one as well.

**Review by johannbg — APPROVED** [review]

**Review by haraldh — APPROVED** [review]

**Pull request merged**